sources/packages/api/src-sql/DisbursementReceipts/Create-disbursement-receipts-table.sql

```diff
@@ -0,0 +1,64 @@
+--Create table sims.disbursement_receipts.
+CREATE TABLE IF NOT EXISTS sims.disbursement_receipts(
+    id SERIAL PRIMARY KEY,
+    batch_run_date DATE NOT NULL,
```

Comment:
is this value from the file? or is the date when the file was processed in our system?